src/main/resources/openapi/sqs/Models/RejectedReasonItem.yaml

```diff
@@ -0,0 +1,43 @@
+title: RejectedReasonItem
+description: Contains elector friendly text and ERO's preference if the reason should be included in comms.
+type: object
+version: 1.0.0
+properties:
+  electorReason:
+    description: Elector friendly reason text. This text is curated and managed within EROP
+    type: string
+    maxLength: 100
+  type:
+    description: |
+      An enum value that can be used by EMSs to add additional business logic if needed.
+      * identity-not-confirmed
+      * signature-is-not-acceptable
+      * fraudulent-application (will be passed with includeInComms:false)
+      * not-registered-to-vote
+      * not-eligible-for-reserved-polls
+      * dob-not-provided
+      * incomplete-application
+      * proxy-limits
+      * proxy-not-registered-to-vote
+      * other-reject-reason (will be passed with includeInComms: false)
+    type: string
+    enum:

```

Comment:
Would it be simpler for this to be a string without any enum restriction. This means we can add new ones with fewer changes and the external facing sync is designed that this should be transparent. Should reduce the amount of mapping

Reply:
Should I be removing the Enum restriction in the mapping in Postal/Proxy apis also then? Otherwise I think we'll get an enum passed through still